install/launch_nomad_jobs.sh

```diff
@@ -13,7 +13,15 @@ nomad_server_private_ip=$(jq -r '.nodes[] | select(has("nomad_server")) | .priva
 node_ssh_username=$(jq -r '.ssh_username' config.json)
 web_ingress_ssh_username=$(jq -r '.nodes[] | select(has("web_ingress")) | .web_ingress.ssh_username' config.json)
 
-sql_server_database_password=$(jq -r '.nodes[] | select(has("sql_server")) | .sql_server.database_password' config.json)
+self_hosted_sql_server=$(jq -r 'any(.nodes[]; has("sql_server"))' config.json)
+if [ $self_hosted_sql_server ]; then
+    sql_server_database_password=$(jq -r '.nodes[] | select(has("sql_server")) | .sql_server.database_password' config.json)
+    sql_server_private_ip=$(jq -r '.nodes[] | select(has("sql_server")) | .private_ip' config.json)
+    sql_uri="postgresql://modelbazaaruser:${sql_server_database_password}@${sql_server_private_ip}:5432/modelbazaar"
+else
+    sql_uri=$(jq -r '.sql_uri' config.json)
```

Comment:
what if the mentioned sql_uri doesn't exist, should we check that exists? assuming users are dumb.??

Reply:
Its definitely users responsibility to make sure what mentioned in readme about sql_uri, but do you think having that check helps?

Reply:
Yeah good point, I'll add a check in setup.sh